compiler/optimizer/Inliner.cpp

```diff
@@ -5011,6 +5011,18 @@ bool TR_InlinerBase::inlineCallTarget2(TR_CallStack * callStack, TR_CallTarget *
             _disableTailRecursion = true;
          }
       }
+   else if (comp()->getOption(TR_FullSpeedDebug) && tif->crossedBasicBlock())
```

Comment:
Does'nt this need to be done whenever "involuntary OSR" is needed ? 
I agree that FSD is the only case in which we do involuntary OSR today but wanted to ask if a more general check here would be better so that this code "just works" in some future (not FSD) scenario where involuntary OSR would be used.